solidity/test/helpers/MathUtils.js

```diff
@@ -20,6 +21,11 @@ function ceilSqrt(n) {
     return Decimal(n).sqrt().ceil().toFixed();
 }
 
+function poweredRatio(a, b, exp) {
```

Comment:
We generally prefer arrow notation (const powerRation = (a, b, exp) =>) in js as well